src/System.Private.ServiceModel/src/Resources/System.Private.ServiceModel.rd.xml

```diff
@@ -29,6 +29,10 @@
         <!-- ServiceContractAttribute on a type makes it available for Reflection -->
         <Type Name="ServiceContractAttribute" Dynamic="Required All" >
           <AttributeImplies Dynamic="Required All" />
+          <!-- Type in CallbackContract named property requires reflection -->
+          <Method Name=".ctor">
+             <TypeParameter Name="CallbackContract" Dynamic="Required All" />
+          </Method>
         </Type>
         <!-- ExceptionDetail requires serialization for FaultException -->
```

Comment:
@mconnew -- good suggestion.  It does work.  I also removed the try/catch because no exception occurs in our test usage.  If we find any usage where an exception occurs, we'll get PNSE and can deal with that as a new issue.